cpu/esp32/Kconfig

```diff
@@ -29,14 +37,17 @@ config CPU_MODEL_ESP32_WROOM_32
 config CPU_MODEL_ESP32_WROVER
```

Comment:
`HAS_ESP_SPI_RAM ` is now a feature that is already provided by `ESP32_WROVER*` modules. Thus
```
    select HAS_ESP_SPI_RAM
```
can be removed from board definitions that use these modules as CPU:
```
boards/esp32-ethernet-kit-v1_0/Kconfig
boards/esp32-ethernet-kit-v1_1/Kconfig
boards/esp32-ethernet-kit-v1_2/Kconfig
boards/esp32-wemos-lolin-d32-pro/Kconfig
boards/esp32-wrover-kit/Kconfig
```
We need it in `boards/esp32-ttgo-t-beam/Kconfig` because it is not provided by the CPU module but as separate module. But, the second `select HAS_ESP_SPI_RAM` should be removed.